main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"go1f/pkg/server"
+	"os"
+
+	"github.com/mariya-goncharenko/go_final_project_my/pkg/db"
+)
+
+func main() {
+	dbFile := os.Getenv("TODO_DBFILE")
+	if dbFile == "" {
+		dbFile = "scheduler.db"
+	}
+
+	if err := db.Init(dbFile); err != nil {
+		fmt.Println("Ошибка при подключении к БД:", err)
```

Comment:
Если так написать, то можно не использовать exit.
```suggestion
		log.Fatal("Ошибка при подключении к БД:", err)
```

Reply:
Исправила

Reply:
Ожидалось, что будет так
![image](https://github.com/user-attachments/assets/02c48d77-f8f1-4df9-b175-57dae5cec9ba)